nncf/common/tensor_new/functions.py

```diff
@@ -0,0 +1,150 @@
+
+
+def all(a: Tensor, axis: Optional[Union[int, Tuple[int]]] = None) -> Tensor:  # pylint: disable=redefined-builtin
+    """
+    Test whether all tensor elements along a given axis evaluate to True.
+
+    :param a: The input tensor.
+    :param axis: Axis or tuple of axes along which to count non-zeros. Default is None,
+       meaning that non-zeros will be counted along a flattened version of a.
+    :return: A new boolean or tensor is returned unless out is specified,
+      in which case a reference to out is returned.
+    """
+    return tensor_func_dispatcher("all", a, axis=axis)
+
+
+def allclose(a: Tensor, b: Tensor, rtol: float = 1e-05, atol: float = 1e-08, equal_nan: bool = False) -> Tensor:
+    """
+    Returns True if two arrays are element-wise equal within a tolerance.
+
+    :param a: The first input tensor.
+    :param b: The second input tensor.
+    :param rtol: The relative tolerance parameter, defaults to 1e-05.
+    :param atol: The absolute tolerance parameter, defaults to 1e-08.
+    :param equal_nan: Whether to compare NaN`s as equal. If True,
+      NaN`s in a will be considered equal to NaN`s in b in the output array.
+      Defaults to False.
+    :return: True if the two arrays are equal within the given tolerance, otherwise False.
+    """
+    return tensor_func_dispatcher("allclose", a, b, rtol=rtol, atol=atol, equal_nan=equal_nan)
+
+
+def any(a: Tensor, axis: Optional[Union[int, Tuple[int]]] = None) -> Tensor:  # pylint: disable=redefined-builtin
+    """
+    Test whether all tensor elements along a given axis evaluate to True.
+
+    :param a: The input tensor.
+    :param axis: Axis or tuple of axes along which to count non-zeros. Default is None,
+       meaning that non-zeros will be counted along a flattened version of a.
+    :return: A new boolean or tensor is returned unless out is specified,
+      in which case a reference to out is returned.
+    """
+    return tensor_func_dispatcher("any", a, axis=axis)
+
+
+def count_nonzero(a: Tensor, axis: Optional[Union[int, Tuple[int]]] = None) -> Tensor:
+    """
+    Counts the number of non-zero values in the tensor input.
+
+    :param a: The tensor for which to count non-zeros.
+    :param axis: Axis or tuple of axes along which to count non-zeros. Default is None,
+       meaning that non-zeros will be counted along a flattened version of a.
+    :return: Number of non-zero values in the tensor along a given axis.
+      Otherwise, the total number of non-zero values in the tensor is returned.
+    """
+    return tensor_func_dispatcher("count_nonzero", a, axis=axis)
+
+
+def is_empty(a: Tensor) -> Tensor:
+    """
+    Return True if input tensor is empty.
+
+    :param a: The input tensor.
+    :return: True is tensor is empty, otherwise False.
+    """
+    return tensor_func_dispatcher("is_empty", a)
+
+
+def isclose(a: Tensor, b: Tensor, rtol: float = 1e-05, atol: float = 1e-08, equal_nan: bool = False) -> Tensor:
```

Comment:
Minor. Please use consistent naming
```python
is_empty()
is_close()
```
or
```
isempty()
isclose()
```
I think first variant is better

Reply:
Selected the second variant, like numpy style https://numpy.org/doc/stable/reference/generated/numpy.isclose.html